config/katello.migrations/180109093420-hammer-ssl-cert.rb

```diff
@@ -0,0 +1,5 @@
+if answers['foreman::cli'].is_a? Hash # If user has already specified options, set the SSL CA file
+  answers['foreman::cli']['ssl_ca_file'] = '/etc/pki/katello/certs/katello-server-ca.crt'
```

Comment:
I wonder if it would be a good practice to extract this from the certs answers or if that'd create more problems because it might not be available yet.

Reply:
Did Kafo change? foreman::cli is at the top level of the installer, so I think it has to be.  The certs are hardcoded like this for all the modules :-(

Reply:
It didn't, but I wonder if we could use answers to get this value and reduce duplication. Looks like it's done in code (https://github.com/theforeman/puppet-certs/blob/60fad4d1057fca79b718c43d16d633be75e3d42d/manifests/init.pp#L110) so it's not going to work anyway.